analytics/filesystem/file_module_test.go

```diff
@@ -23,6 +24,8 @@ func TestAmpObject_ToJson(t *testing.T) {
 	}
 	if aoJson := jsonifyAmpObject(ao); strings.Contains(aoJson, "Transactional Logs Error") {
 		t.Fatalf("AmpObject failed to convert to json")
+	} else {
+		fmt.Println(aoJson)
```

Comment:
```go
assert.JSONEq(t, expectedAoJson, aoJson, "incorrect json")

....


const expectedAoJson = `
{
  "type": "/openrtb2/amp",
  "Status": 200,
  "Errors": [],
  "Request": null,
  "AuctionResponse": {
    "id": ""
  },
  "AmpTargetingValues": {},
  "Origin": "",
  "StartTime": "0001-01-01T00:00:00Z",
  "HookExecutionOutcome": null
}`

```

Reply:
The print line was accidentally left in. I removed it.

Reply:
Do you think we need to check positive scenarios too?